src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -167,11 +187,12 @@ def activate(
                     activation_db_logger,
                 )
 
+            activation_db_logger.flush()
         except DeactivationException:
             msg = f"Activation {activation.name} is disabled"
-            instance.status = ActivationStatus.STOPPED
```

Comment:
@hsong-rh Shouldn't we have a return here since the finally will run and we suppress the exception there, but if the activation worker is running ahead of the default worker they would both write to the DB.